src/lib/form-field/form-field.html

```diff
@@ -8,24 +8,31 @@
     <div class="mat-input-infix mat-form-field-infix">
       <ng-content></ng-content>
 
-      <span class="mat-input-placeholder-wrapper mat-form-field-placeholder-wrapper">
+      <span class="mat-form-field-label-wrapper">
```

Comment:
I'm sure people are targeting those classes with custom styles, lets leave them for now with a TODO to remove them at the next major.